tokenizers/src/pre_tokenizers/unicode_scripts/unicode.rs

```diff
@@ -0,0 +1,2093 @@
+// Generated by modified Perl script at https://github.com/google/sentencepiece/blob/master/data/gen_unicode_scripts_code.pl
```

Comment:
This file seems to be a duplicate of `unicode_scripts/scripts.rs`

Reply:
Yes, I kinda messed up my rebases